sdk/healthinsights/azure-healthinsights-radiologyinsights/README.md

```diff
@@ -0,0 +1,142 @@
+# Azure Cognitive Services Health Insights Radiology Insights client library for Python
+[Health Insights][health_insights] is an Azure Applied AI Service built with the Azure Cognitive Services Framework, that leverages multiple Cognitive Services, Healthcare API services and other Azure resources.
+
+[Radiology Insights][radiology_insights_docs] is a model that aims to provide quality checks as feedback on errors and inconsistencies (mismatches) and ensures critical findings are identified and communicated using the full context of the report. Follow-up recommendations and clinical findings with measurements (sizes) documented by the radiologist are also identified.
+
+## Getting started
+
+### Prequisites
+
+- Python 3.8 or later is required to use this package.
+- You need an [Azure subscription][azure_sub] to use this package.
+- An existing Cognitive Services Health Insights instance.
+
+For more information about creating the resource or how to get the location and sku information see [here][cognitive_resource_cli].
+
+### Installing the module
+
+```bash
+python -m pip install azure-healthinsights-radiologyinsights
+```
+This table shows the relationship between SDK versions and supported API versions of the service:
+
+| SDK version | Supported API version of service |
+|-------------|----------------------------------|
+| 1.0.0b1     | 2024-01-17-preview               |
```

Comment:
This API version doesnt look right to me, shouldnt it be the 2023-09-01-preview version? Link to TypeSpec: https://github.com/Azure/azure-rest-api-specs/blob/f1fbb15d46f2c451a34a2d0e37b26c5ff745db29/specification/ai/HealthInsights/HealthInsights.OpenAPI/service.tsp#L25

Reply:
changed